cfme/fixtures/post_appliance_upgrade.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+
+def pytest_addoption(parser):
+    group = parser.getgroup("cfme")
+    group.addoption(
+        "--post-upgrade",
```

Comment:
I think this is a good name for the marker for tests themselves, but I'm thinking `--upgrade-appliance` would be a better CLI trigger for this. Then we have `--update-appliance` to trigger OS update, and `--upgrade-appliance` to trigger application upgrade